scripts/jenkins-build.bat

```diff
@@ -68,11 +68,32 @@ cmake -G "Ninja" ^
 cmake --build . || exit /b
 cmake --build . --target install || exit /b
 
+REM Build Java
 cd "%WORKSPACE%"
+cd source
+call mvn clean install
```

Comment:
Nods, I started with `mvn clean install || exit /b` (b699e37ddb35d7ed817a5fb08fab0830659195b9)  but this did not work as the execution stops after the first call. The `call` is solution I could find based on the various threads on the topic e.g. http://stackoverflow.com/questions/4877099/call-batch-file-from-cygwin-breaks-after-1st-command.